README.md

```diff
@@ -2,61 +2,84 @@
+<a href="https://github.com/snapcrafters/jenkins/actions/workflows/sync-version-with-upstream.yml"><img src="https://github.com/snapcrafters/jenkins/actions/workflows/sync-version-with-upstream.yml/badge.svg"></a>
+<a href="https://github.com/snapcrafters/jenkins/actions/workflows/release-to-candidate.yaml"><img src="https://github.com/snapcrafters/jenkins/actions/workflows/release-to-candidate.yml/badge.svg"></a>
+<a href="https://github.com/snapcrafters/jenkins/actions/workflows/promote-to-stable.yml"><img src="https://github.com/snapcrafters/jenkins/actions/workflows/promote-to-stable.yml/badge.svg"></a>
 </p>
 
 ## Install
 
-    sudo snap install jenkins
-    xdg-open http://localhost:8080
+```shell
+snap install jenkins --classic
+```
 
 ([Don't have snapd installed?](https://snapcraft.io/docs/core/install))
 
-Check the status:
+<p align="center">Published for <img src="https://raw.githubusercontent.com/anythingcodes/slack-emoji-for-techies/gh-pages/emoji/tux.png" align="top" width="24" /> with :gift_heart: by Snapcrafters</p>
+
+## How to contribute to this snap
+
+Thanks for your interest! Below you find instructions to help you contribute to this snap.
+
+The general workflow is to submit pull requests that merges your changes into the `candidate` branch here on GitHub. Once the pull request has been merged, a GitHub action will automatically build the snap and publish it to the `candidate` channel in the Snap Store. Once the snap has been tested thoroughly, we promote it to the `stable` channel so all our users get it!
+
+### Initial setup
+
+If this is your first time contributing to this snap, you first need to set up your own fork of this repository.
+
+1. [Fork the repository](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) into your own GitHub namespace.
+2. [Clone your fork](https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository), so that you have it on your local computer.
+3. Configure your local repo. To make things a bit more intuitive, we will rename your fork's remote to `myfork`, and add the snapcrafters repo as `snapcrafters`.
+
+   ```shell
+   git remote rename origin myfork
+   git remote add snapcrafters https://github.com/snapcrafters/jenkins.git
+   git fetch --all
+   ```
+
+### Submitting changes in a pull request
+
+Once you're all setup for contributing, keep in mind that you want the git information to be all up-to-date. So if you haven't "fetched" all changes in a while, start with that:
+
+```shell
+git fetch --all -p
+```
 
-    systemctl status snap.jenkins.jenkins.service
+Now that your git metadata has been updated you are ready to create a bugfix branch, make your changes, and open a pull request.
 
-Set a custom configuration:
+1. All pull requests should go to the stable branch so create your branch as a copy of the stable branch:
 
-    cat config.xml | sudo /snap/bin/jenkins.config
```

Comment:
We shouldn't remove this info, imo. Put it under its own headline after `## Install`. Something like `## Snap Configuration`